projects/kserve/testing/config.yaml

```diff
@@ -208,21 +232,47 @@ ci_presets:
     tests.e2e.llm_load_test.args.concurrency: [1, 2, 4, 8, 16, 32, 64, 96, 128]
     export_artifacts.enabled: true
 
-  # ---
-
-  caikit_single_model_gating:
-    extends: [gating, 10min, e2e_gpu, e2e_perf]
-    kserve.model.directory_prefix: caikit_tgis
-    tests.e2e.models: [flan-t5-small-gpu, flan-t5-large-gpu, bloom-560m, mpt-7b-instruct2]
-    matbench.lts.horreum.test_name: rhoai-kserve-single
-    matbench.lts.opensearch.index: rhoai-kserve-single
+  # --
+  # single-model vLLM
+  # --
+  vllm_cpt_single_model_gating_light:
+    extends: [raw, e2e_gpu, vllm]
+    tests.e2e.models:
+    - name: phi-2
+      testing:
+        size: small
+        max_concurrency: 64
+    tests.e2e.matbenchmark.enabled: true
+    tests.e2e.llm_load_test.args.concurrency: [1, 2, 4, 8, 16, 32, 64, 96, 128]
 
-  caikit_single_model_gating_light:
-    extends: [caikit_single_model_gating, 5min, gating_rehearsal]
+  vllm_cpt_single_model_gating:
+    extends: [e2e_perf, 3min, raw, use_intlab_minio, vllm] # TODO gating
+    tests.e2e.models:
+    - name: japanese-llama-2-7b-instruct
+      testing:
+       size: small
+       max_concurrency: 64
+    - name: llama-2-13b-chat-hf
+      testing:
+        size: small
+        max_concurrency: 32
+    - name: mpt-7b-instruct2
+      testing:
+        size: small
+        max_concurrency: 64
+    - name: llama-2-70b-chat-hf
+      testing:
+        size: large
+    - name: codellama-34b-instruct-hf
+      testing:
+        size: large
```

Comment:
Did you miss to add the phi-2 model?

Reply:
Or is phi-2 just for running a smaller model? I don't think its part of the watsonx list, right?